src/hooks/useImageUpload.ts

```
@@ -0,0 +1,78 @@
+"use client"
+
+import { useState } from "react"
+import { useAuth } from "../contexts/AuthContext"
+
+interface UploadResponse {
+    success: boolean
+    error: string | null
+    data: any
```

Biome (1.9.4)

[error] Unexpected any. Specify a different type.

any disables many type checking rules. Its use should be avoided.

(lint/suspicious/noExplicitAny)